src/makernote_int.cpp

```diff
@@ -227,11 +227,9 @@ namespace Exiv2 {
         return 0;
     }
 
-    const byte OlympusMnHeader::signature_[] = {
-        'O', 'L', 'Y', 'M', 'P', 0x00, 0x01, 0x00
-    };
+    constexpr byte OlympusMnHeader::signature_[]; // Required in C++11, but deprecated in C++17
 
-    uint32_t OlympusMnHeader::sizeOfSignature()
```

Comment:
Can you move these functions into the header? Since constexpr functions are implicitly marked as inline, there's little use keeping them in the source file.

Reply:
The only usage would be to hide implementation details, but since we are moving the definition of the literal to the header file, your comment make sense. 👍